sway-lib-core/src/ops.sw

```diff
@@ -550,10 +485,186 @@ impl OrdEq for u8 {
 impl OrdEq for b256 {
 }
 
+pub trait Shiftable {
```

Comment:
moved Shiftable trait down so it can use other traits above. This is already tracked in an issue.